docs/src/components/TableOfContent/TableOfContent.css

```diff
@@ -10,6 +10,14 @@
   display: inline-block;
 }
 
+.toc-link-small {
+  text-decoration: none;
+  font-size: var(--font-size-s);
+  color: var(--inverse);
+  padding: var(--spacing) 0;
+  display: inline-block;
+}
+
```

Comment:
Please create a separate class only for `font-size`. All other properties are same as previous class `toc-link`

Reply:
done